dspace/config/log4j2.xml

```diff
@@ -29,7 +29,7 @@
 
         >
             <Layout type='PatternLayout'
-                    pattern='%d %-5p %c @ %m%n'/>
+                    pattern='%d %-5p $${ctx:correlationID:-unknown} $${ctx:requestID:-unknown} %c @ %m%n'/>
```

Comment:
Unfortunately, I forgot that this `log4j2.xml` file is [filtered by Ant](https://github.com/DSpace/DSpace/blob/main/dspace/src/main/config/build.xml#L355), which means we MUST escape these dollar signs. I tried this out locally, and below is the correct syntax, as Ant transforms a `$$` into a literal `$`.
```
<!-- NOTE: this pattern uses four dollar signs to escape property substitution by the Ant filter
     process used to install this file. Once installed, only two dollar signs will exist. -->
<Layout type='PatternLayout'
        pattern='%d %-5p $$$${ctx:correlationID:-unknown} $$$${ctx:requestID:-unknown} %c @ %m%n'/>
```

Reply:
Hi @tdonohue, thanks for pointing out the missing escape. I fixed the bug and also added the logging.server.include-after-request property to enable/disable log after request.